server.js

```diff
@@ -25,10 +24,11 @@ let state = null;
 const production = process.env.NODE_ENV === "production";
 const port = process.env.PORT || 1337;
 const wsPort = process.env.WS_PORT || 2448;
-const resetData = process.env.npm_config_reset_data;
-const app = next({ dev: !production, quiet: false });
+const app = next({ dev: !production, dir: __dirname, quiet: false });
 const nextRequestHandler = app.getRequestHandler();
 
+const path = require("path");
```

Comment:
Shouldn't this be `import path from 'path';`?

Reply:
Thanks for catching this, fixed w/ latest commit.